src/Components/App/FillRequestsPage.tsx

```diff
@@ -85,7 +86,9 @@ export const FillRequestsPage = (props: FillRequestsPageProps) => {
                 </>
             )}
 
-            <div className="action-button-row">
+            <div className="action-button-row" style="gap: 10px;">
+                {sent && <NextRequestButton setPage={props.setPage} />}
```

Comment:
If you only show this if `sent === true` this will never be a skip button. Is this on purpose? Because I would prefer it differently.